### PR TITLE
Add empty adjusted Xcode project for JourneySquad App.

### PR DESCRIPTION
**Pull request by kirpichonok**
# What 

Added initial Xcode project to start development of JourneySquad application. There are set up minimum deployment target, supported devices, modes and platform, project version.

# Why

It's part of work on [issue #1](https://github.com/ios-course/link-team-project/issues/1). 

# How 

Added folder with Xcode project to the root directory.

### There are set up next values:

- Base SDK - iOS 

- iOS Deployment Target - 16.0

- Current Project Version - 0.0.0

- Supported Destinations  - iPhone

- Minimum Deployments - iOS 16.0

- iPhone Orientation - Portrait only

- Identity: 
  -  Version - 0.0.0
  - Build - 0.0.0

### Added SwiftLint script to build phase

    export PATH="$PATH:/opt/homebrew/bin"
    if which swiftlint > /dev/null; then
      swiftlint --config ../.swiftlint.yml
    else
      echo "warning: SwiftLint not installed, download from https://github.com/realm/SwiftLint"

### CODE REVIEW

**Review by YuliaNastiushkina — COMMENTED** [review]
I cannot build project, I have next issue:
`One of the path in DEVELOPMENT_ASSET_PATH does not exist.`

Does anyone else have this issue, or is it a mistake on my side?

**Comment by kirpichonok** [comment]
Yes, I have the same error. I'm figuring out.

**Comment by kirpichonok** [comment]
I've updated the project. Please, try again.

**Review by YuliaNastiushkina — APPROVED** [review]
LGTM

File: JourneysSquad/JourneysSquad.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -0,0 +1,375 @@
...
+				INFOPLIST_KEY_UIApplicationSupportsIndirectInputEvents = YES;
+				INFOPLIST_KEY_UILaunchScreen_Generation = YES;
+				INFOPLIST_KEY_UISupportedInterfaceOrientations = UIInterfaceOrientationPortrait;
+				INFOPLIST_KEY_UISupportedInterfaceOrientations_iPad = "UIInterfaceOrientationLandscapeLeft UIInterfaceOrientationLandscapeRight UIInterfaceOrientationPortrait UIInterfaceOrientationPortraitUpsideDown";
+				IPHONEOS_DEPLOYMENT_TARGET = 16.0;
+				LD_RUNPATH_SEARCH_PATHS = (
+					"$(inherited)",
+					"@executable_path/Frameworks",
+				);
+				MARKETING_VERSION = 0.0.0;
```
Comment (dersim-davaod) [inline JourneysSquad/JourneysSquad.xcodeproj/project.pbxproj:307]: You have set up the marketing version in the target, but did not do this in the project. Why?

**Review by dersim-davaod — CHANGES_REQUESTED** [review]
LGTM, except the version settings.

**Review by levvaa — APPROVED** [review]
LGTM

File: JourneysSquad/JourneysSquad.xcodeproj/project.pbxproj (hunk 1 above)
Comment (kirpichonok) [reply JourneysSquad/JourneysSquad.xcodeproj/project.pbxproj]: It's my mistake. I wanted to set up versions only for PROJECT and not to set up versions for TARGETS. 
Fixed

**Comment by kirpichonok** [comment]
I set up the target's marketing version to **$MARKETING_VERSION** and the project version to **CURRENT_PROJECT_VERSION**. They will be updated with PROJECT's project and marketing versions.

**Review by dersim-davaod — APPROVED** [review]
LGTM

File: JourneysSquad/JourneysSquad/Assets.xcassets/AccentColor.colorset/Contents.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
```
[the remark this answers is not in the document]
Comment (kirpichonok) [reply JourneysSquad/JourneysSquad/Assets.xcassets/AccentColor.colorset/Contents.json:1]: May be we can delete the content, but we need this directory to build the project. I don't see the need to delete this file and then to think how to push this dir to repo.

**Pull request merged**